app/Actions/Action.js

```diff
@@ -0,0 +1,11 @@
+class LocalInventoryActions {
```

Comment:
why make it a class ? You can make SET_INVENTORY_ID a const separately so it can be used and setInventoryId a normal function. The aim of hooks useReducer and all is to get rid of classes. And move to complete functional programming.

Keeping constants separately would help manage them.

Reply:
why make it a class ?
I am not using class in real meaning just wanna modularise related properties and methods in a single wrapper (wrapper maybe class or object) in future I'll add Auth0 and Upload inventory classes etc.

Reply:
Hmm we can discuss that later should be easy to tackle that as well let me push change for this and later we can discuss for auth0 and upload classes keeping it modularised. How to achieve that might be little tricky now with useReducer and all but still lets try to do that without classes.